internal/server/server.go

```diff
@@ -69,10 +69,15 @@ func createSOMASServer(
 			DeerPopulation:  foraging.CreateDeerPopulationModel(gameConfig.ForagingConfig.DeerHuntConfig),
 			ForagingHistory: forageHistory,
 			IIGOHistory:     map[uint][]shared.Accountability{},
-			SpeakerID:       shared.Team1,
-			JudgeID:         shared.Team2,
-			PresidentID:     shared.Team3,
-			CommonPool:      gameConfig.InitialCommonPool,
+			IIGORolesBudget: map[shared.Role]shared.Resources{
```

Comment:
Could we possibly wrap the SpeakerID, JudgeID and PresidentID in the same struct as their budgets, it seems a bit awkward with the budgets grouped together and the ID's separate. Maybe smth like IIGORolesStatus{ID, budget ...}

Reply:
This doesn't seem to be a very quick fix, as the RoleIDs are used in several places. 
I can try to implement it now if you see it as a high priority, if not I can try fixing it at a later stage in a separate PR?